src/routes/profile/[id=profile_id]/+layout.server.ts

```diff
@@ -0,0 +1,34 @@
+import type { ProfileData } from '../../../lib/types/profile-data.type'
+import { BACKEND_BASE_URL } from '$env/static/private'
+import { redirect, type Cookies } from '@sveltejs/kit'
+
+export async function load({
+  fetch,
+  params: { id },
```

Comment:
```suggestion
  params,
```
Creo que estás especificando el tipo dos veces

Reply:
En esa línea estoy destructurando la propiedad id del objeto params, no es tipado